contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -0,0 +1,817 @@
+
+        ///
+        /// Decision
+        ///
+
+        /// @dev The id of the last withdrawal request that should be finalized as the result
+        /// of applying this oracle report. The zero value means that no requests should be
+        /// finalized.
+        uint256 lastWithdrawalRequestIdToFinalize;
+
+        /// @dev The share/ETH rate with the 10^27 precision (i.e. the price of one stETH share
+        /// in ETH where one ETH is denominated as 10^27) used for finalizing withdrawal requests
+        /// up to (and including) the one passed in the lastWithdrawalRequestIdToFinalize field.
+        /// Must be set to zero if lastWithdrawalRequestIdToFinalize is zero.
+        uint256 finalizationShareRate;
+
+        /// @dev Whether, based on the state observed at the reference slot, the protocol should
+        /// be in the bunker mode.
+        bool isBunkerMode;
+
+        ///
+        /// Extra data — the oracle information that can be processed asynchronously in chunks
+        /// after the main data is processed. The oracle doesn't enforce that extra data attached
+        /// to some data report is processed in full before the processing deadline expires or a
+        /// new data report starts being processed, but enforces that no processing of extra data
+        /// for a report is possible after its processing deadline passes or a new data report
+        /// arrives.
+        ///
+        /// Extra data is an array of items, each item being encoded as follows:
+        ///
+        ///    3 bytes    2 bytes     27 bytes
+        /// | itemIndex | itemType | itemPayload |
+        ///
+        /// itemIndex is a 0-based index into the extra data array;
+        /// itemType is the type of extra data item;
+        /// itemPayload is the item's data which interpretation depends on the item's type.
+        ///
+        /// Two types of items are supported:
+        ///
+        /// itemType=EXTRA_DATA_TYPE_STUCK_VALIDATORS: stuck validators by node operator.
+        /// itemPayload format:
+        ///
+        ///   3 bytes        8 bytes            16 bytes
+        /// | moduleId | nodeOperatorId | totalStuckValidators |
+        ///
+        /// itemType=EXTRA_DATA_TYPE_EXITED_VALIDATORS: exited validators by node operator.
+        /// itemPayload format:
+        ///
+        ///   3 bytes        8 bytes             16 bytes
+        /// | moduleId | nodeOperatorId | totalExitedValidators |
+        ///
+        /// Extra data array should be sorted in ascending order by the following compound key:
+        ///
+        /// (itemType, moduleId, nodeOperatorId)
+        ///
+        /// Extra data array can be passed in different formats, see below.
+        ///
+
+        /// @dev Format of the extra data. Currently, only the EXTRA_DATA_FORMAT_LIST=0 is
+        /// supported. See the constant defining a specific extra data format for more info.
+        uint256 extraDataFormat;
```

Comment:
do we need `uint256` here? maybe `uint8`?

Reply:
we don't need to pack this